### ISSUE TITLE
Configure error with gmvapich/MVAPICH2

### ISSUE THREAD

**Issue by verdurin** [hpcugent/easybuild-easyconfigs#3395]
Trying to build `MVAPICH2-2.2b-GCC-4.9.3-2.25.eb`, as specified by `gmvapich2-2016a`, I see this error during the configure step:

```
checking for library containing ibv_open_device... -libverbs
checking infiniband/verbs.h usability... yes
checking infiniband/verbs.h presence... yes
checking for infiniband/verbs.h... yes
configure: checking checking for infiniband umad installation...
checking for library containing umad_init... no
configure: infiniband libumand not found
checking for ibv_open_xrc_domain... yes
checking whether to enable hybrid communication channel... no
configure: checking for RDMA CM support...
checking for library containing rdma_create_event_channel... no
configure: RDMA CM support disabled
configure: checking for Hardware multicast support enabled...
checking for library containing mad_get_field... no
checking infiniband/mad.h usability... no
checking infiniband/mad.h presence... no
checking for infiniband/mad.h... no
configure: error: 'infiniband/mad.h not found. Please retry with --disable-mcast'
 (at easybuild/tools/run.py:397 in parse_cmd_output)
== 2016-07-28 15:03:09,402 easyblock.py:2367 WARNING build failed (first 300 chars): cmd " ./configure --prefix=/camp/apps/eb/software/MVAPICH2/2.2b-GCC-4.9.3-2.25  --with-rdma=gen2  --with-thread-package=pthreads --enable-fast --enable-shared --enable-sharedlibs=gcc --enable-static --enable-f77 --enable-fc --enable-cxx " exited with exitcode 1 and output:
./configure: line 4615: te
== 2016-07-28 15:03:09,405 easyblock.py:270 INFO Closing log for application name MVAPICH2 version 2.2b
```

**Comment by arif-ali** [comment]
Hi @verdurin,

I encountered this problem as well. I believe it would be wise to add the following into the eb config

```
osdependencies = [('libibmad-devel','libibverbs-devel')]
```

So on the build machine, in this situation, we need to install `libibmad-devel`

**Comment by verdurin** [comment]
Thanks for the pointer, @arif-ali 

I've actually addressed this by updating the easyconfigs for `libibmad` and `libibumad`, and adding them to `MVAPICH2`:

#3588 
#3587

**Comment by boegel** [comment]
follow-up in #3587 (merged) and #3589 (to be merged)
